### DISCUSSION TITLE
Is it possible to use Sql Server SQL Graph as a rdf / triplestore?

### DISCUSSION THREAD

**Discussion by wstaelens** [dotnetrdf/dotnetrdf#521] [Q&A]
I've discovered that SQL Server has SQL Graph capabilities: 
https://learn.microsoft.com/en-us/sql/relational-databases/graphs/sql-graph-architecture?view=sql-server-ver16
https://codingsight.com/how-to-make-use-of-sql-server-graph-database-features/

Is it possible to use this as an RDF / Triplestore using dotnetrdf? If possible, any code samples available?

**Comment by kal** [comment]
Thats an interesting suggestion. I think it would be possible to do it, but there would be quite a lot of work involved in the implementation - especially to do things like import/export of RDF and SPARQL query efficiently. I think you could get a toy implementation working in a few weeks of development, but making it robust and scalable would be a lot more work.
